### PR TITLE
add basic time chart components with animation

### PR DESCRIPTION
**Pull request by ceenell**
This:
- fixes spacing among symbols in legend
- adds a bar plot showing the proportion of sites in each category
- adds a line timeseries showing the proportion of sites in each category that can double as a time component
- uses gsap to set up a timeline animation
- uses D3 to drive daily animation through gwls across all chart
- contributes to some page styling
- resolves the issue with svg optimization that was stripping `<g>` elements during build in `vue.config.js` (related to https://github.com/USGS-VIZLAB/gw-conditions/issues/18)

### CODE REVIEW

**Comment by ceenell** [comment]
Looks like this:

Screen Shot 2021-09-20 at 8 55 17 AM

**Comment by ceenell** [comment]
The latest commits address https://github.com/USGS-VIZLAB/gw-conditions/issues/34
I added a css grid for positioning with two layouts - the default being stacked in mobile and a break at 1024px for desktop. Each element is in a container that is assigned a grid-area, and css is used to align svgs within their assigned area.

Desktop: 
image

Mobile:
image

File: src/components/GWL.vue [hunk 1]
```diff
@@ -1,18 +1,25 @@
...
   <section>
-    <div id="title-container">
-      <h1>U.S. Groundwater Conditions</h1></div>
-    <div>
-
-</div>
+    <div id="grid-container">
+      <div id="title-container">
+        <h2 class="title" id="title-main">U.S. Groundwater Conditions</h2>
+        <h3 class="title" id="title-sub">The low down on flow down low</h3>
```
Comment (lindsayplatt) [inline src/components/GWL.vue]: 🤣 and 🤞 that it stays!

File: src/components/GWL.vue [hunk 2]
```diff
@@ -22,28 +29,74 @@ export default {
...
+      //x:  0,
+      sites_list: null,
+      //t2: null,
+
+      // style for timeline
+      button_color: "grey",
+      button_hilite: "black",
+
+
+       // TODO: derive from pipeline. inputs are months, day sequence nested w/ key
```
Comment (lindsayplatt) [inline src/components/GWL.vue:79]: 👍 noted

File: public/perc_df.json [hunk 3]
```diff
@@ -0,0 +1 @@
...:351,\"n_sites\":347,\"n_sites_total\":1525,\"perc\":0.2275},{\"wyday\":352,\"n_sites\":344,\"n_sites_total\":1521,\"perc\":0.2262},{\"wyday\":353,\"n_sites\":342,\"n_sites_total\":1519,\"perc\":0.2251},{\"wyday\":354,\"n_sites\":347,\"n_sites_total\":1518,\"perc\":0.2286},{\"wyday\":355,\"n_sites\":356,\"n_sites_total\":1516,\"perc\":0.2348},{\"wyday\":356,\"n_sites\":359,\"n_sites_total\":1513,\"perc\":0.2373},{\"wyday\":357,\"n_sites\":371,\"n_sites_total\":1512,\"perc\":0.2454},{\"wyday\":358,\"n_sites\":361,\"n_sites_total\":1512,\"perc\":0.2388},{\"wyday\":359,\"n_sites\":355,\"n_sites_total\":1508,\"perc\":0.2354},{\"wyday\":360,\"n_sites\":361,\"n_sites_total\":1508,\"perc\":0.2394},{\"wyday\":361,\"n_sites\":367,\"n_sites_total\":1506,\"perc\":0.2437},{\"wyday\":362,\"n_sites\":367,\"n_sites_total\":1505,\"perc\":0.2439},{\"wyday\":363,\"n_sites\":365,\"n_sites_total\":1504,\"perc\":0.2427},{\"wyday\":364,\"n_sites\":370,\"n_sites_total\":1503,\"perc\":0.2462},{\"wyday\":365,\"n_sites\":372,\"n_sites_total\":1502,\"perc\":0.2477},{\"wyday\":366,\"n_sites\":344,\"n_sites_total\":1499,\"perc\":0.2295},{\"wyday\":367,\"n_sites\":134,\"n_sites_total\":771,\"perc\":0.1738}]}]"]
```
Comment (lindsayplatt) [inline public/perc_df.json:1]: Is this something the pipeline needs to produce and push to S3?

**Review by lindsayplatt — COMMENTED** [review]
Not sure if you are looking for a review of your Vue/D3 code, but I would recommend asking Hayley if you would like a detailed review of that. I don't think I have the right qualifications. 

In terms of content, 👍

**Comment by lindsayplatt** [comment]
Looks like there may be some invisible boundary that is cutting this peak off: 

image

File: public/perc_df.json (hunk 3 above)
Comment (ceenell) [reply public/perc_df.json:1]: Yeah, I am still not able to do it to the public bucket

Comment (lindsayplatt) [reply public/perc_df.json:1]: Is the data created via pipeline, but you just can't build to the public bucket?

**Comment by ceenell** [comment]
👍🏼 Adding @hcorson-dosch as a reviewer

File: public/perc_df.json (hunk 3 above)
Comment (ceenell) [reply public/perc_df.json:1]: I just added it to the pipeline but it currently exports to the public folder and gets ignored.  https://github.com/USGS-VIZLAB/gw-conditions/pull/33/commits/f535d8cb671beedc6762e6a56b840961d977b23f

**Pull request merged**